HSF-TN/draft-2016-PROJ/draft-HSF-TN-2016-PROJ.md

```diff
@@ -20,18 +20,18 @@ Afterwards we provide a checklist of the proposals you may want to use for your
 
 
 
 # Supporting Developers and Contributors
 The following sections discuss points mainly relevant for project developers and potential new contributors.
 
 ## Code repository
-The first requirement for an open-source project is fully versioned code in a *public repository*. The code should be accessible in anonymous read-only mode by everybody. Services like [GitHub](https://github.com) or [GitLab](https://gitlab.com) provide it for free. In addition efforts like [hepforge](https://www.hepforge.org/) or labs like CERN or DESY may host HEP-specific packages. Services supporting a clone plus *merge-request/pull-request workflow* can be helpful to attract new contributors.
+The first requirement for an open-source project is fully versioned code in a *public repository*. The code should be accessible in anonymous read-only mode by everybody. Services like [GitHub](https://github.com) or [GitLab](https://gitlab.com) provide it for free. In addition efforts like [hepforge](https://www.hepforge.org/) or labs like CERN or DESY may host HEP-specific packages. Services supporting a clone plus *merge-request/pull-request workflow* can be helpful to attract new contributors, as it is the current de-facto standard of open software development.
 
-## License
-The code and software provided should be properly licensed in order to be able to use code provided by others, and to allow people to re-use, update, or improve the software you provide. The [HSF technical note](http://hepsoftwarefoundation.org/technical_notes.html) *HSF-TN-2016-01* (*Software Licence Agreements HSF Policy Guidelines*) discusses various options. This is one of the topics that is typically ignored at the beginning of a project and hard to fix afterwards.
+## License and Copyright
+The ownership and copyright of the code has to be well defined and understood. In a second step, the code and software provided should be properly licensed in order to be able to use code provided by others, and to allow people to re-use, update, or improve the software you provide. The [HSF technical note](http://hepsoftwarefoundation.org/technical_notes.html) *HSF-TN-2016-01* (*Software Licence Agreements HSF Policy Guidelines*) discusses various options. This is one of the topics that is typically ignored at the beginning of a project and hard to fix afterwards.
```

Comment:
Note that ascribing copyright to one of the large HEP labs is generally a good option?